vcs/VTKPlots.py

```diff
@@ -14,6 +14,11 @@
 import vcsvtk
 
 
+def _makeEven(val):
+    if (val & 0x1):
+        val -= 1
```

Comment:
ok i think you're right if anything it's more likely to go above scree size than to go to zero. So I won't touch it.